pgjdbc/src/main/java/org/postgresql/util/PSQLState.java

```diff
@@ -93,6 +93,8 @@
   SYSTEM_ERROR("60000"),
   IO_ERROR("58030"),
 
+  RELATION_DOES_NOT_EXIST("42P01"),
```

Comment:
`UNDEFINED_TABLE("42P01"),` and should be in the 42 section.

https://www.postgresql.org/docs/current/errcodes-appendix.html

Reply:
fixed